docs/cmdline-opts/write-out.d

```diff
@@ -67,6 +67,10 @@ The http method used in the most recent HTTP request (Added in 7.72.0)
 .B num_connects
 Number of new connects made in the recent transfer. (Added in 7.12.3)
 .TP
+.B num_headers
+The number of response headers in the most recent request (restarted at each
+ redirect).
```

Comment:
`(Added in 7.73.0)` (assuming we manage to merge this before the feature window closes which might be hard, it is more likely to be targeted for inclusion in 7.74.0)